cpp: update related to v0.2.1 (#23397)
DataDog/documentation: content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/cpp.md

## Instrument your application

Here is an example application that can be used for testing `dd-trace-cpp`.

> This application creates a tracer instance with the default settings and generates a trace with two spans, which will be reported under the service name `my-service`.

Reviewer: ```suggestion
This application creates a tracer instance with the default settings and generates a trace with two spans, which is reported under the service name `my-service`.
```

cmake --build build -j
cmake --install build
```

> By default `cmake --install` places the shared library and public headers into the appropriate system directories (e.g., `/usr/local/[...]`).

Reviewer: ```suggestion
By default, `cmake --install` places the shared library and public headers into the appropriate system directories (for example, `/usr/local/[...]`).
```
